scripts/ci/conda/upload.sh

```diff
@@ -4,17 +4,19 @@
 if [ -z "${ANACONDA_TOKEN+x}" ]
 then
     echo "Anaconda token is not set, not uploading"
-    exit 0;
+    exit 1;
 fi
 
 ls
 pwd
 find .
 
-if [ -z "${ANACONDA_TOKEN}" ]
-then
-    echo "Anaconda token is empty, not uploading"
-    exit 0;
+set PACKAGE_PATTERN="*pdal*.conda"
```

Comment:
As far as I can see your current conda-build version still produces .bz2 files. For some reason GDAL must use a more recent one than PDAL's one

Reply:
Thanks. We want it to be producing `.conda` files going forward. I will update